e2e/app-with-domain/app_with_domain_test.go

```diff
@@ -288,6 +288,7 @@ var _ = Describe("App With Domain", func() {
 			var resp *http.Response
 			var fetchErr error
 			urls := strings.Split(route.URL, " or ")
+			expectedResponseBody := "This is Copilot express app for admin"
```

Comment:
Do we want to hit the root path `/` or `/me` as well? I feel like this only proves that port `3002` is accepting the traffic.